examples/test_sqlite.py

```diff
@@ -0,0 +1,20 @@
+"""Minimum working example (MWE) to use SQLiteDataConverter
+"""
+
+from gnn_reco.data.sqlite_dataconverter import SQLiteDataConverter
+import pandas as pd
+
+def main():
+    """Main script function."""
+    paths = ['/groups/icecube/asogaard/data/i3/i3_to_sqlite_workshop_test/level7_v02.00']
+    mode  = 'oscNext'
+    pulsemap = 'SRTInIcePulses'
+    gcd_rescue = '/groups/icecube/stuttard/data/oscNext/pass2/gcd/GeoCalibDetectorStatus_AVG_55697-57531_PASS2_SPE_withScaledNoise.i3.gz'
+    outdir = '/groups/icecube/asogaard/temp/sqlite_test'
+    db_name = 'gnn-reco_test'
+    workers = 2
+
+    SQLiteDataConverter(paths, mode, pulsemap, gcd_rescue, outdir, db_name, workers)
+
+if __name__ == '__main__':
+    main()
```

Comment:
I would personally prefer to move `if __name__` checks into `SQLiteDataConverter` such that the user doesnt have to worry about this.

Reply:
It is good practice that all scripts that are meant to be used as executables include such a `__name__` guard to avoid executing the script on import (see https://google.github.io/styleguide/pyguide.html#317-main). However, I have also tried to run the script without this guard in place with the same results, meaning that this is strictly not something users _have_ to worry about, although it would be good form if they did. :)  Also, this check serves a different purpose that the `if __name__ == ...` check in `SQLiteDataConverter` and so it won't be a choice of one or the other.